src/db/error.cr

```diff
@@ -17,4 +17,7 @@ module DB
     def initialize(@connection)
     end
   end
+
+  class Rollback < Exception
+  end
```

Comment:
DB::Error are used when the origin of the exception was an interaction with the database.
Rollback is initiated by the user.